admin-api-gateway/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM node:10.15 AS base
+
+WORKDIR /usr/app
+
+ENV PORT 3000
+
+
+
+FROM base
+
+COPY package*.json ./
+
+RUN npm ci --only=production
+
+COPY . .
+
+CMD ["npm", "start"]
```

Comment:
I could be wrong, but I see no "start" script in package.json

Reply:
It is part of `npm help start`
> NAME
>        npm-start - Start a package
> ...
> DESCRIPTION
>        This runs an arbitrary command specified in the package's "start" prop-
>        erty of its "scripts" object. If no "start" property  is  specified  on
>        the "scripts" object, it will run node server.js.